src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionValue.cs

```diff
@@ -119,7 +119,7 @@ internal MotionValue(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, c
         /// Get or set the value as PropertyValue type.
```

Comment:
could probably drop the word "type", e.g. "Get or set the PropertyValue - a value of a property" maybe

Reply:
Fix done.